Macros/e3 Includes/e3_Classes_Magician.inc

```diff
@@ -63,192 +63,397 @@ sub event_petWeapons(line, ChatSender)
 /if (${Debug} || ${Debug_MAG}) /echo <== event_petWeapons -|
 /return
 
+
+|#event armPets "<#1#> Pet Weapon#*#"
+|#event armPets "<#1#> Arm Pets"
+|#event armPets "<#1#> Arm Pets #2#"
```

Comment:
You have removed the ability for a BC tell to work, lets not remove functionality that was there before.

Reply:
Fixed now. Just had to add a couple lines to parse parameters when its a bc.